src/CoursePageResources/messages.js

```diff
@@ -0,0 +1,82 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  'course-authoring.pages-resources.heading': {
```

Comment:
nit: I think the idea is to give is a more friendly name here. 

I think the 'id' needs to be globally unique, but this key in the dictionary does not need to be.
So I would recommend keeping them simple, so, 'heading' instead of 'course-authoring.pages-resources.heading'.

Reply:
OK, I'll remove "course-authoring.pages-resources." from the dictionary keys but leave the id fields alone.